src/redux/user/user.sagas.js

```diff
@@ -395,4 +428,5 @@ export default function* userSaga() {
   yield takeEvery(UPDATE_USER, handleUpdateUser);
   yield takeEvery(SEND_CONFIRMATION_EMAIL, handleSendConfirmation);
   yield takeEvery(GET_USER_ORDERS, handleGetUserOrders);
+  yield takeEvery(GOOGLE_USER, handleGoogleUserLogin);
```

Comment:
You need to use a more understandable type name for this action (like **LOGIN_USER_BY_GOOGLE** or **LOGIN_BY_GOOGLE**)

Reply:
Fixed